go.sum

```diff
@@ -511,6 +511,8 @@ github.com/open-telemetry/opentelemetry-proto v0.0.0-20200211051721-ff5f19c6217d
 github.com/open-telemetry/opentelemetry-proto v0.0.0-20200211051721-ff5f19c6217d/go.mod h1:PMR5GI0F7BSpio+rBGFxNm6SLzg3FypDTcFuQZnO+F8=
 github.com/open-telemetry/opentelemetry-proto v0.0.0-20200227223449-a65b867510c0 h1:g8x0BMEHMKlLYJEYj0SucilB796mVMcef/xjnd/rH8E=
 github.com/open-telemetry/opentelemetry-proto v0.0.0-20200227223449-a65b867510c0/go.mod h1:PMR5GI0F7BSpio+rBGFxNm6SLzg3FypDTcFuQZnO+F8=
+github.com/open-telemetry/opentelemetry-proto v0.0.0-20200308012146-674ae1c8703f h1:o21WlujGsrjoN3+n99AflASF/K0S6+SOLOj2O9nNplc=
+github.com/open-telemetry/opentelemetry-proto v0.0.0-20200308012146-674ae1c8703f/go.mod h1:PMR5GI0F7BSpio+rBGFxNm6SLzg3FypDTcFuQZnO+F8=
```

Comment:
Please run `go mod tidy` to prune old references if you didn't already.

Reply:
done.